proposals/simd/SIMD.md

```diff
@@ -104,6 +104,73 @@ operations. This means that any subnormal operand is treated as 0, and any
 subnormal result is rounded to 0. Note that this differs from WebAssembly
 scalar floating-point semantics which require correct subnormal handling.
 
+# JavaScript API and SIMD Values
+
+Accessing WebAssembly module imports or exports containing SIMD Type from JavaScript will throw.
+
+### Module Function Imports
+
+Calling an imported function from JavaScript when the function arguments or result is of type v128 will cause the host function to immidiately throw a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror).
+
+### Module Global Imports
+
+If a global import has its global_type as v128, throw a WebAssembly.LinkError.
+
+### Exported Globals
+
+If an exported global is of type v128, throw a WebAssembly.LinkError
+
+### Exported Function Exotic Objects
+
+Invoking the [[Call]] method of an Exported Function Exotic Object when the function type of its [[Closure]] has an argument or result of type v128 will cause the host function to immidiately throw a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror).
+
+
+## WebAssembly Module Instatiation
+
+Instantiating a WebAssembly Module from a Module moduleObject will throw a LinkError exception, when module_imports(moduleObject.[[Module]]) has an externtype of the form global mut valtype and valtype is v128.
```

Comment:
This is legal as long as the imported object's type is `WebAssembly.Global` and the global's valtype is `v128`.